ambry-replication/src/main/java/com.github.ambry.replication/ReplicationManager.java

```diff
@@ -392,7 +401,7 @@ public void updateTotalBytesReadByRemoteReplica(PartitionId partitionId, String
   public long getRemoteReplicaLagInBytes(PartitionId partitionId, String hostName, String replicaPath) {
```

Comment:
Could you change this function name to be `getRemoteReplicaLagFromLocalInBytes` to be consistent with the changes to `RemoteReplicaInfo`?